CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.79.7
```

Comment:
Not sure if it is intended, 1.79.6 has not been released.

Reply:
Ah, thanks